generators/server/generator.ts

```diff
@@ -105,6 +105,8 @@ export default class JHipsterServerGenerator extends BaseApplicationGenerator {
   projectVersion;
 
   async beforeQueue() {
+    const { skipClient, clientFramework } = this.jhipsterConfigWithDefaults;
```

Comment:
At this point skipClient will alway be false and clientFramework will alway be angular for new projects.

Reply:
Oops, for microservices skipClient will alway be true and clientFramework will always be no.